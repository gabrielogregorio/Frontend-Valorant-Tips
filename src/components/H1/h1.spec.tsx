import { render, screen } from "@testing-library/react"
import { H1 } from '.'

describe('<H1 />', () => {
  it('should render h1', () => {
    render(<H1 title="Testando componente" />)
    expect(screen.getByRole('heading', {name: /Testando componente/i})).toBeInTheDocument()
  })

  it('should match snapshot', () => {
    const { container } = render(<H1 title="Testando componente" />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
