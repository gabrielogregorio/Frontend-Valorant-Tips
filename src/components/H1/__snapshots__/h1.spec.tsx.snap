// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<H1 /> should match snapshot 1`] = `
<h1>
  Testando componente
</h1>
`;
